### DISCUSSION TITLE
Replace `Mapbox` with `MapLibre` across the code base

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre-native#318] [General]
Should we replace `Mapbox` with `MapLibre` across the code base? This would probably lead to breaking changes in all platforms so we might want to bundle that with other breaking changes...

**Comment by hy9be** [comment]
Took a look today and found 2k+ Mapbox occurrences in *.cpp files. I suggest we make a phased plan and replace them incrementally. 

Btw, @wipfli is there a way for me to know our current test coverage? High test coverage should make changes like this less daunting.

**Reply by wipfli** [reply]
I don't know how to measure the test coverage. But currently we are not running render and query tests in CI, see #350. These tests helped us a lot to find bugs in the TypeScript migration in GL JS.

**Reply by hy9be** [reply]
I have "integrating with codecov" on my plan. Will prioritize it after solving more burning issues (e.g. CI with Apple Silicon runners, and updating the documentation)
